releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -2,11 +2,29 @@
 
 [Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
+## 4.0.0-preview4 - Jul 19, 2024
+
+### Enhancements
+
+* Feature flags declared in the [.NET Feature Management schema](https://github.com/microsoft/FeatureManagement-Dotnet/blob/main/schemas/FeatureManagement.Dotnet.v1.0.0.schema.json) will also be recognized when they appears together with [Microsoft Feature Management schema](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureManagement.v2.0.0.schema.json). [#470](https://github.com/microsoft/FeatureManagement-Dotnet/pull/470)
+
+* Feature manager now publish telemetry through `Activity`. If telemetry is enabled for a feature flag, whenever the feature flag is evaluated, feature manager will start an `Activity` and add an `ActivityEvent` with tags contains feature evaluation information.  [#455](https://github.com/microsoft/FeatureManagement-Dotnet/pull/455)
+
+### Breaking Changes
+
+* Variant and telemetry are no longer supported for [.NET Feature Management schema](https://github.com/microsoft/FeatureManagement-Dotnet/blob/main/schemas/FeatureManagement.Dotnet.v1.0.0.schema.json). Please use [Microsoft Feature Management schema](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureManagement.v2.0.0.schema.json) to declare variant feature flag instead. [#421](https://github.com/microsoft/FeatureManagement-Dotnet/pull/421)
```

Comment:
This doesn't make sense. On one hand, you are saying this is a breaking change to .NET Feature Management schema  and on the other hand you are saying .NET Feature Management schema is an enhancement.

Reply:
How about just not mention we now support using .NET and microsoft schema at the same time?

Reply:
Yes. Previously, both .NET schema and Microsoft schema support variant. So the feature management will only respect one of the schema. If microsoft schema can be found, .NET schema will be ignored.

But now, we removed variant & telemetry from .NET schema. The .NET provider used to write feature flag in .NET schema but now it has to write variant feature flag in microsoft schema. We had a discussion about this. We don't want to break the users who are using old version of FM (e.g. v2.6) which only recognizes .net schema. So the .NET provider will convert non-variant feature flag to .NET schema and variant feature flag to microsoft schema. This change requires feature management to respect both schemas if both schemas can be found in configuration. (I used to consider this as an enhancement.)

From the customer side, they may not need to know this change. What they need to do is just variant is removed from .NET schema.

Reply:
> If you were using earlier preview versions of this package and configuration files to define variant feature flags, they are no longer supported in the .NET Feature Management schema. Instead, use the Microsoft Feature Management schema.

So does what I suggested above work because what really breaks is when people define variant feature flags in files?

Reply:
> does what I suggested above work

Yes. Updated.